src/web/get-started.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Build a web app with Dart
+description: Get started writing web apps in Dart.
+---
+
+Follow these steps to start using Dart to develop **web-only** apps.
+If you want to write a **multi-platform** app, then
+[try Flutter.]({{site.flutter}}/web)
+
+If you haven't yet, first learn the basics of Dart,
+by following the [Introduction to Dart](/language).
+
+Once you feel comfortable with the basics, you can
+follow the steps below to create a small web app with Dart.
+
+## 1. Install Dart {#install-dart}
+
+{% include get-sdk.md %}
```

Comment:
This include doesn't make sense here, because there's no mention of dartpad before this point. 

I think it'd be best to just edit the includes file to be agnostic of dartpad anyway (hopefully will make use of it a lot in future tutorial content 🤞), so I'll do that in another PR and you can leave this

Reply:
Thank you!!